src/views/virtualmachines/details/tabs/scheduling/components/VirtualMachineSchedulingLeftGrid.tsx

```diff
@@ -40,13 +40,14 @@ const VirtualMachineSchedulingLeftGrid: React.FC<VirtualMachineSchedulingLeftGri
   const { t } = useKubevirtTranslation();
   const { createModal } = useModal();
 
-  const isMachineNotLiveMirgation = !!vm?.status?.conditions?.find(
-    ({ type, status }) => type === 'LiveMigratable' && status === 'False',
+  // TODO: Can "LiveMigratable" condition have any more values than "True" and "False"?
```

Comment:
I think it might be better to use here the `useSingleNodeCluster` hook to be able to use the function you wrote `isLiveMigratable` which is a very good solution to your comment IMO, since when you check if 
`status === 'True'` you don't really care if there's other values which is exactly what we want :)

Reply:
@avivtur I had more dilemmas in this place of the code, and not because I wouldn't know how to use `isLiveMigratable` here (at first I wanted to use it here), but because here it is about checking if we can edit Descheduler field. And if you look into the popover text, there's:
_Note: if VirtualMachine have LiveMigration=False condition, edit is disabled._ 
So I assumed if it is not `"False"`, then Descheduler should be editable, no matter the VM status or if there's just a one node in the cluster. And as we know, this all is checked within `isLiveMigratable`. So I assumed I better won't use `isLiveMigratable` in here + I will keep checking for "False", not "True". But if there are more options than True and False, the check for editability of the Descheduler may not be quite correct, IMO.

Reply:
I understand what you're saying that's some good thinking, I'm not sure if there's any more values other than 'True' or 'False' I'm afraid but I think that the actual descheduler is also making the proper checks in the backend so that's why checking for the LiveMigration=False condition as the helper text suggest will be enough